src/main/java/org/openmrs/standalone/TomcatManager.java

```diff
@@ -79,6 +79,7 @@ public TomcatManager(String contextName, int port) {
 		
 		// create engine
 		Engine engine = container.createEngine();
+        engine.setService(container);
```

Comment:
It's kind of bizarre with the new tomcat API that i have to set the service,  i would assume that container.createEngine would automatically set it, may be they have reasons why they don't